packages/block-editor/src/components/plain-text/content.scss

```diff
@@ -1,11 +1,27 @@
-.block-editor-plain-text {
+.block-library-html__edit .block-editor-plain-text.components-base-control {
+	padding: 0 !important;
```

Comment:
There's an unfortunate `padding: 12px !important` that we're trying to override here. Ideally, it wouldn't be there in the first place, but that might be work for another PR.